### PR TITLE
Query with irrelevant wrapper

### PR DESCRIPTION
**Pull request by DoctorVoid**

### CODE REVIEW

File: src/schema-helpers/query-with-irrelevant-wrapper.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { QueryIrrResult } from '@enigmatis/utills';
+import { Model } from 'mongoose';
+import { InnerModelType } from '../../../mongo-driver/src/types';
+import { RepositoryModel } from '../model-creator';
+
+export const QueryWithIrrelevant = async(
+    model: Model<InnerModelType<any>>,
+    result: any[],
+    dataVersion: number | undefined,
+): Promise<QueryIrrResult> => {
```
Comment (yarinvak) [inline src/schema-helpers/query-with-irrelevant-wrapper.ts]: Irr->IrrelevantEntities

File: src/schema-helpers/query-with-irrelevant-wrapper.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    dataVersion = dataVersion || 0;
+    const irrelevant = await model.find(
+        {
+            _id: { $nin: result.map(x => x._id) },
+            deleted: { $in: [true, false] },
```
Comment (yarinvak) [inline src/schema-helpers/query-with-irrelevant-wrapper.ts]: is it necessary?

File: src/schema-helpers/query-with-irrelevant-wrapper.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { QueryIrrelevantResult } from '@enigmatis/utills';
+import { Model } from 'mongoose';
+import { InnerModelType } from '../../../mongo-driver/src/types';
```
Comment (osher-sade) [inline src/schema-helpers/query-with-irrelevant-wrapper.ts]: Is there any shorter import path available?

**Review by osher-sade — APPROVED** [review]

**Pull request merged**